src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let carCost = 0;
```

Comment:
Consider using `const` instead of `let` for `carCost` since its value is not reassigned after the initial calculation. This adheres to checklist item 3, which prefers `const` over `let` to avoid unintentional changes.